src/main/flight/mixer.c

```diff
@@ -386,7 +386,7 @@ void initEscEndpoints(void)
     default:
         if (feature(FEATURE_3D)) {
             disarmMotorOutput = flight3DConfig()->neutral3d;
-            motorOutputLow = PWM_RANGE_MIN;
+            motorOutputLow = motorConfig()->minthrottle;
```

Comment:
This is wrong, and will limit the maximum negative motor output when flying 3D.

Reply:
It will not limit it. It will allow the maimum negative motor output to be changed with the configurator. see issue #5101.

Reply:
Ok this should be fine.

Reply:
@I-Fly-Anything Actually I realise this might need to be mincommand instead of minthrottle perhaps?

Reply:
I think either would work. In non 3D, minthrottle is used for lowest PWM to keep the motor spinning, and mincommand is used to stop the motors. I like using minthrottle because it is analogous to maxthrottle. Would there be an issue if minthrottle were lower than mincommand? Ideally I think minthrottle and mincommand should be set to the same value for 3D quads.

Reply:
Yeah probably for 3D it should be the same.

Reply:
Ok, I see your point about wanting to make this adjustable. However, it should be a new, independent parameter, and neither `minthrottle` nor `mincommand`, since both of these have to be tuned for non-3D flying, and should not be re-used for something that should be adjustable independently. As an alternative, 'mirroring' `max_throttle` on center throttle would give a symmetrical maximum negative thrust, not sure if that is what is wanted / needed/

Reply:
I like the idea of a separate parameter, but I don't know enough about betaflight and its configurator to set that up. I don't think mirroring `max_throttle` is the best solution.

Reply:
The parameter should not be integrated into the configurator straight away, it should first be CLI only until it's been tested and established that this is actually the right way. The idea behind re-using `max_throttle` (which is definitely better than re-using something like `minthrottle` that is used for something unrelated) is to start with a minimum implementation and see if this solves the problem at hand.

Reply:
I agree with @mikeller but support the idea of a separate variable for the 3D limits, I've always just dealt with what's there but also knew that in edge cases like this it would be ideal to be able to set values for each independently. I also think that 3D configuration beyond standard should be left to CLI only at least for a while, if people can't understand what they are changing then they shouldn't be changing it.

I know that there haven't been a full set of separate variables for 3D parameters in the past but it would likely be handy to set that up and give them meaningful names, especially if we start looking at some extras around smoothing of reversing e.t.c.

Reply:
@mikeller I am not sure how to implement these changes to create a new parameter.

Reply:
@I-Fly-Anything: It's not that hard, it just needs to be done in a number of places. Have a look at how it's done for `deadband3d_low`, and try to replicate this:

```
$ grep -r deadband3d_low src/main
src/main/fc/rc_controls.h:    uint16_t deadband3d_low;                // min 3d value
src/main/fc/rc_controls.c:    .deadband3d_low = 1406,
src/main/interface/msp.c:        sbufWriteU16(dst, flight3DConfig()->deadband3d_low);
src/main/interface/msp.c:        flight3DConfigMutable()->deadband3d_low = sbufReadU16(src);
src/main/interface/settings.c:    { "3d_deadband_low",            VAR_UINT16 | MASTER_VALUE, .config.minmax = { PWM_PULSE_MIN, PWM_RANGE_MIDDLE }, PG_MOTOR_3D_CONFIG, offsetof(flight3DConfig_t, deadband3d_low) },
src/main/flight/mixer.c:        deadbandMotor3dLow = flight3DConfig()->deadband3d_low;
$ 
```

Reply:
@mikeller, thanks for the info. I will be working on this.